contracts/MarketOracle.sol

```diff
@@ -14,104 +14,128 @@ interface IOracle {
 /**
  * @title Market Oracle
  *
- * @dev Provides the aggregated exchange rate data onchain using data from a whitelisted
- *      set of market source contracts.
- *      Exchange rate is the TOKEN:TARGET rate.
+ * @dev Provides the aggregated rate data onchain using data from a whitelisted
```

Comment:
Style note: I wonder if we should call this "MedianWhitelistOracle" or something more generic.

I don't think any logic here will be specific to our usage of it in the monetary policy. If so, the comments would be generic also.

Reply:
I like MedianOracle,  I am avoiding changing the filenames as Git doesn't do well with that. We can do the name change as the last step.